sss/sss_table.py

```diff
@@ -155,17 +305,13 @@ def read_file(file):
         print('This file cannot be read' + ' ' + file.split('/')[-1])
     return df, file
 
-
-def pre_check(df):
-    print(df.dtypes)
-
-
 def check_extra_columns(df):
     """
     This function takes in a pandas dataframe and adds a boolean column(s)
 
-    Specifically, this function checks if broadband_&_cell_phone,
-    and health_care are included the dataframe
+    Specifically, this function checks if the arpa, health care,    
+    broadband_&_cell_phone, and are included the dataframe. We 
+    update the boolean column to represent whether the columns are in dataframe. 
```

Comment:
The Returns section for this docstring is actually wrong. It says there's only one dataframe returned but there are actually four! (sorry, github won't let me put this comment on the appropriate line because that line wasn't modified).